package.json

```diff
@@ -8,7 +8,7 @@
   "license": "MPL-2.0",
   "homepage": "https://github.com/iron-fish/ironfish-api",
   "engines": {
-    "node": "16.x"
+    "node": "17.x"
```

Comment:
Curious why this version bump - 16 is an LTS release and the same version we use for the main codebase

Reply:
Yeah, `Production applications should only use Active LTS or Maintenance LTS releases.`

Also can we split this version bump out into another PR since it might be controversion?

Reply:
Yeah. I'll split this out - was testing some stuff with Prisma and should have left it out.